### ISSUE TITLE
World map - faraway chunks always have height = 1

### ISSUE THREAD

**Issue by whitebyte** [anegostudios/VintageStory-Issues#598]
**Game Version:** 1.13.4
**Platform:** Linux (KDE Neon)
**Modded:** No

### Description
Coordinates tooltip in the worldmap incorrectly shown height = 1 for faraway chunks (maybe for never visited, but I'm not sure). 

### Screenshots

![zindex1](https://user-images.githubusercontent.com/7704186/94471758-5eca2c80-01d2-11eb-8413-6847f898fc7f.png)

**Comment by Craluminum2413** [comment]
It looks like it shows actual height on the worldmap only for loaded chunks

**Comment by copygirl** [comment]
True, that is likely the cause. But there's no reason it couldn't remember the height at any given position, just like it remembers the last block (or at least color) at that location.

**Comment by maltiez2** [comment]
Addressed in MR1321
